timeseries/src/autogluon/timeseries/models/gluonts/models.py

```diff
@@ -15,6 +15,7 @@
     from gluonts.model.seq2seq import MQCNNEstimator, MQRNNEstimator
     from gluonts.model.simple_feedforward import SimpleFeedForwardEstimator
     from gluonts.model.transformer import TransformerEstimator
+    from gluonts.model.tft import TemporalFusionTransformerEstimator
```

Comment:
very cool. is the fix for TFTs backported to v0.9?

Reply:
ah sorry I was asking about this: "Thanks to Lorenzo's fix from https://github.com/awslabs/gluonts/issues/2264 we can add TFT to AutoGluon (the model should work both on CPU and GPU)." but I guess that's `hybridize=False`.